runtime/doc/test-coverage.md

```diff
@@ -74,4 +74,5 @@ Test Scenario| Test Name | Runtime Error Code
 Check validation of DPE structure after a warm/update reset and ensure that validation fails if the DPE's SRAM bytes are maliciously edited | N/A | N/A
 Verify the RT Journey PCR on a warm reset | N/A | N/A
 Check that the RT Journey PCR was updated correctly on update reset | N/A | N/A
-Check if disable attestation was called when the mailbox is executing a command during a warm reset | N/A | N/A
\ No newline at end of file
+Check if disable attestation was called when the mailbox is executing a command during a warm reset | N/A | N/A
+Check that measurements in the measurement log are added to DPE upon initializing drivers | N/A | N/A
```

Comment:
Can you open a github issue for this as well?

Reply:
Done https://github.com/chipsalliance/caliptra-sw/issues/1094